tools/sci-old-mess/fonts.py

```diff
@@ -72,14 +71,33 @@ def convert(stream):
         bim.paste(im, box=((idx % grid_size) * w, int(idx / grid_size) * h))
             # im.save(f'try/char_{idx:03d}.png')
         # print(off, stream.tell())
-    bim.save('font.png')
+    return bim
+
+# Actual color indices are {64, 0, 110, 48, 95}
```

Comment:
What's this color thing?
I don't understand.
Is this TraduSCI format, or something that's relevant for Sierre SCI?

Reply:
Actually the changes in this files were added to this branch by mistake.
I have modified this old tool to allow making preview images of several fonts at once.
The specific line in question is that the fonts are made specifically with those color indices and I have manually selected grayscale shades with high contrast to ease the preview